gr-blocks/lib/mute_impl.cc

```diff
@@ -36,41 +37,36 @@ mute_impl<T>::mute_impl(bool mute)
 {
     this->message_port_register_in(pmt::intern("set_mute"));
     this->set_msg_handler(pmt::intern("set_mute"),
-                          [this](pmt::pmt_t msg) { this->set_mute_pmt(msg); });
+                          [this](pmt::pmt_t msg) { this->mute_message_handler(msg); });
```

Comment:
can we make the signature of this `const pmt::pmt& msg` rather than `pmt::pmt_t msg`?

Background is that you're really just "passing through" values, but pass-by-value on `pmt_t` is actually a reference count increase, and that comes with mutex overhead!

(Technical background 2: `pmt_t` is actually a `std::shared_ptr<pmt_base>`, which means it does a lot of cross-thread synchronized book-keeping behind the scenes when it's copied and when a copy is destroyed).